lib/kubernetes-deploy/runner_task_config_validator.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+module KubernetesDeploy
+  class RunnerTaskConfigValidator < TaskConfigValidator
+    DEFAULT_VALIDATIONS += %i(
```

Comment:
I can't believe this is valid Ruby. Should we have a class method for adding new validations instead of doing so upsetting things to constants.

Reply:
I just sent a commit with a change to make the addition of those two new validations happen during `initialize`; not sure yet if that's the best way. I'll check the class method idea!